.github/workflows/autopkgtest.yaml

```diff
@@ -34,6 +34,14 @@ jobs:
     - name: Checkout
       uses: actions/checkout@v2
 
+    # The version number needs to be the same as the one in
+    # https://github.com/pistacheio/pistache/blob/debian/changelog
+    - name: Create .orig tarball
+      run: |
+        cd ..
+        repo_name=$(echo ${{ github.repository }} | cut -d / -f 2)
+        tar  --gzip --exclude-vcs -cvf pistache_0.0.003.orig.tar.gz "$repo_name"
```

Comment:
Mostly good, but we have a minor problem here. The version probably shouldn't be hardcoded into the tarball name in this file. Instead it should be parsed from `version.txt`. Actually, even better, shouldn't the build environment generate the tarball (e.g. CMake)?

Reply:
Unfortunately no, as it has to be in sync with what is in the debian directory, regardless of the contents of the "upstream" files.
But thinking about it, I could directly parse the version from the debian directory itself

Reply:
Yeah, I think that's the way to do it. awk(1) / sed(1) are your friends.

Reply:
Also, to clarify, neither CMake nor Meson are safe to use to generate the tarball because its contents have to be exactly identical (excluding vcs files) to the ones already present in the checked out directory. So, to make things simple and robust, I simply archive the checked out dir so that I'm sure that I don't have extra files nor missing ones.

Reply:
Ah yes, I recall now. We talked about this before when we were discussing the GNU Autotools versus other build environments.

Reply:
> Yeah, I think that's the way to do it. awk(1) / sed(1) are your friends.

Luckily `dpkg-parsechangelog` already exists :D

Reply:
Your Debian kung fu is growing stronger every day.